core/services/ocr2/plugins/llo/config/config.go

```diff
@@ -39,13 +39,19 @@ type PluginConfig struct {
 	// KeyBundleIDs maps supported keys to their respective bundle IDs
 	// Key must match llo's ReportFormat
 	KeyBundleIDs map[string]string `json:"keyBundleIDs" toml:"keyBundleIDs"`
+
+	DonID uint32 `json:"donID" toml:"donID"`
 }
 
 func (p *PluginConfig) Unmarshal(data []byte) error {
 	return json.Unmarshal(data, p)
 }
 
 func (p PluginConfig) Validate() (merr error) {
+	if p.DonID == 0 {
+		merr = errors.New("llo: DonID must be specified and not zero")
```

Comment:
Should the errors following this one all be `errors.Join`ed? Many are just assigned to `merr` directly.

Reply:
Good catch, fixed.